cypress/integration/modal_spec.js

```diff
@@ -0,0 +1,34 @@
+describe('Modal Spec', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/').get('.add-post-btn').click();
+  });
+
+  it('Should open a modal upon clicking Add a Post!', () => {
+    cy.get('.modal-content').should('be.visible');
+  });
+
+  it('Should contain an input form for title', () => {
+    cy.get('.post-title').should('be.visible');
+  });
+
+  it('Should contain an input form for description', () => {
+    cy.get('.post-description').should('be.visible');
+  });
+
+  it('Should contain an input to add images', () => {
+    cy.get('.upload__image-wrapper').should('be.visible');
+  });
+
+  it('Should contain an option to remove images', () => {
+    cy.get('.remove-img-btn').should('be.visible');
+  });
+
+  it('Should contain an submit button', () => {
+    cy.get('.post-submit-btn').should('be.visible');
+  });
+
+  it('Submit button should close modal and return to homescreen', () => {
```

Comment:
Nice! This is a good start! Once we add it in, should definitely test for those various user flows around adding an image and seeing the preview, removing and deleting, not submitting until required fields are filled out, etc